yt_dlp/extractor/amazonminitv.py

```diff
@@ -171,22 +104,22 @@ def _real_extract(self, url):
             'subtitles': subtitles,
             'language': traverse_obj(title_info, ('audioTracks', 0)),
             'thumbnails': [{
-                'id': type_,
-                'url': url,
-            } for type_, url in (title_info.get('images') or {}).items()],
-            'description': traverse_obj(title_info, ('description', 'synopsis')),
-            'release_timestamp': int_or_none(try_get(title_info, lambda x: x['publicReleaseDateUTC'] / 1000)),
+                'id': 'imageSrc',
+                'url': title_info.get('imageSrc'),
+            }] if title_info.get('imageSrc') else [],
+            'description': traverse_obj(title_info_, ('description', 'synopsis')),
+            'release_timestamp': int_or_none(try_get(title_info_, lambda x: x['publicReleaseDateUTC'] / 1000)),
             'duration': traverse_obj(title_info, ('description', 'contentLengthInSeconds')),
             'chapters': [{
                 'start_time': credits_time,
                 'title': 'End Credits',
             }] if credits_time else [],
-            'series': title_info.get('seriesName'),
-            'series_id': title_info.get('seriesId'),
-            'season_number': title_info.get('seasonNumber'),
-            'season_id': title_info.get('seasonId'),
+            'series': title_info_.get('seasonName') if is_episode else None,
+            'series_id': title_info.get('seriesId') if is_episode else None,
+            'season_number': title_info.get('seasonNumber') if is_episode else None,
+            'season_id': title_info.get('seasonId') if is_episode else None,
             'episode': title_info.get('name') if is_episode else None,
-            'episode_number': title_info.get('episodeNumber'),
+            'episode_number': title_info.get('episodeNumber') if is_episode else None,
             'episode_id': asin if is_episode else None,
```

Comment:
The tests don't return any values for these fields:
description, thumbnail, duration, series, series_id, season_number, season_id, episode_number, episode_id, language

Reply:
check now.